### DISCUSSION TITLE
Request for prebuilt Sierra or High Sierra image, please

### DISCUSSION THREAD

**Discussion by MatthewLStevens** [utmapp/UTM#5674] [macOS]
I can't get UMT (on M1 Mac, OS 12.6) to create a Sierra or High Sierra VM, even though I'm following all the instructions (vague though they be). The UTM Web site offers prebuilt images of various VMs, including Debian, Linux, Ubuntu and Windows, but nothing for OSX versions.

Does anyone have an image for 10.12 or 10.13 that they'd be willing to share with me?

**Comment by tbussmann** [comment]
Configurations for running macOS / OS X / Mac OS X in UTM with the qemu emulation can be found at https://github.com/adespoton/utmconfigs but your mileage may vary. Recent macOS versions tend to be too slow to use in emulation. If the configurations are not working at all, try older versions of UTM as some changes in later QEMU versions seem to be incompatible with macOS guests currently.

**Reply by MatthewLStevens** [reply]
Thank you, Tobias. I had already downloaded the images from adespoton but I had to experiment with iso images until I found one that worked. I now have High Sierra installed. However, it kept freezing in UTM 4.3.5. As you recommended, I tried an earlier version—4.1.6—and it seems to be working now. As the VM is a backup in case I need to open file format no longer supported on Apple Silicon software, neatness is not important, only "running-ness".

I will later experiment to find the newest UTM version that works without freezing.

**Reply by MatthewLStevens** [reply]
Follow-up: Version 4.1.6 also froze, so I installed 4.0.9, and it is working well.

**Reply by some1new2** [reply]
> I had to experiment with iso images until I found one that worked. 
> 
I'm also having trouble with iso, none recognised on boot. Could you link to a working one?

**Reply by FrankWhiteNYNY** [reply]
Did you ever get it to work? I just got a new G4 Mac Mini and I want to try UTB. When I go to select a ISP disk image that I know is good and works (I have it from Apple and have used it) under CD/DVD it is greyed out. I have tried several other sites to download one that will show up, but they fail. 


Any help would be appreciated.

**Reply by MatthewLStevens** [reply]
I can only reiterate that I was able to get the ISO image called "MacOSHighSierra.iso" (downloaded from [https://archive.org/details/mac-os-high-sierra-iso](https://archive.org/details/mac-os-high-sierra-iso)) to work under UTM 4.0.9 (68) on my M1 Mac Studio under Monterey 12.7.6. I haven't dared update anything lest I break something!

**Reply by FrankWhiteNYNY** [reply]
I have not been successful yet. I did get it to boot x86 Windows 11, it does work for playing old games but it is so slow it is unusable. I'll bet running old OSX will be just as slow?

**Reply by MatthewLStevens** [reply]
Given that operation depends on the host hardware as well as on the efficiency of the guest OS, I suspect you're right. As I've commented before, I keep the VM's handy just in case I need access to a file saved in an old format, so speed is not important to me.

**Comment by MatthewLStevens** [comment]
On 16.10.2024, at 1:24 pm, some1new2 ***@***.***> wrote:

> I had to experiment with iso images until I found one that worked.
> 
> I'm also having trouble with iso, none recognised on boot. Could you link to a working one?
> 
Currently the Internet Archive is down on account of a data breach. When it’s up again, I can find the image I used and send you the link.

**Comment by MatthewLStevens** [comment]
Until the Internet Archive comes back up, I can partly answer your question.

> I had to experiment with iso images until I found one that worked.
> 
> I'm also having trouble with iso, none recognised on boot. Could you link to a working one?
> 
IIRC, no Sierra image will work.

The High Sierra image I use is called MacOSHighSierra.iso (8.28 GB). The Info panel on the file identifies it as having come from 

https://archive.org/details/mac-os-high-sierra-iso <https://archive.org/details/mac-os-high-sierra-iso>

and

https://ia801803.us.archive.org/13/items/mac-os-high-sierra-iso/MacOSHighSierra.iso <https://ia801803.us.archive.org/13/items/mac-os-high-sierra-iso/MacOSHighSierra.iso>

HTH.
